src/components/screens/ScoreSetEditor.vue

```diff
@@ -1303,8 +1337,8 @@ export default {
         delete this.targetGene.targetAccession;
       }
       else if (this.assembly || this.geneName) {
-        this.targetGene.targetAccession.assembly = this.assembly || null
-        this.targetGene.targetAccession.gene = this.geneName?.name || null // Name property on string object array
+        this.targetGene.targetAccession.assembly = this.assemblyDropdownValue
```

Comment:
This is not equivalent to the previous code (not sure if the previous code was correct) as assemblyDropdownValue has .trim() called on it. Are you sure the previous version was incorrect here?

Reply:
These two lines of codes are in addTarget function and getting value of assembly or geneName, that's why I think we still need .trim() to remove leading and trailing whitespaces from the assembly and geneName. Based on that I think we can use this.assemblyDropdownValue and this.geneNameDropdownValue at here either.

Reply:
I did some trials. I think Assembly/HGNC only accepts dropdown menu values. Do I had better change this part back to the previous code? I'm not sure whether in this case we need to care leading and trailing whitespaces or not.

Reply:
fine to leave it as this.assemblyDropdownValue, I don't think the lack of trim() has any advantage.